pkg/operator/ceph/object/objectstore.go

```diff
@@ -752,6 +751,8 @@ func enableRGWDashboard(context *Context) error {
 		DisplayName: &DashboardUser,
 		SystemUser:  true,
 	}
+	// TODO:
+	// Use admin ops user instead!
 	u, errCode, err := CreateUser(context, user)
```

Comment:
is this needed before merge?

Reply:
It's for "future" reference, once we complete the migration.